README.md

```diff
@@ -40,5 +40,27 @@ $ python -m ipykernel install --name great-expectations-workshop
 
 ### Initialization
```

Comment:
Initialization could be: Setting up for part 2 maybe... or something more descriptive like 'intialization new GE project through CLI' or something